app/models/patient.rb

```diff
@@ -115,10 +111,6 @@ def risk_priority
 
   def risk_priority_label
```

Comment:
A thought on this – this method is used in two places currently. One in the [CSV overdue list](https://github.com/simpledotorg/simple-server/blob/master/app/models/appointment.rb#L165) and in the [patient line list](https://github.com/simpledotorg/simple-server/blob/master/app/exporters/patients_exporter.rb#L78l). I was under the impression that it was used in the dashboard as well, but the [dashboard](https://github.com/simpledotorg/simple-server/blob/master/app/views/appointments/index.html.erb#L37) simply shows `High` based on the `high_risk?` method instead of calling this label function.

Since this method now just returns `High` shall we just rely on showing the `High` label in both CSV export + line list by reading off the `high_risk?` boolean? This will also relieve the model of seemingly view-related concerns.

What do you think?

Reply:
Ooh good eye, I'll take a look at this method again.